src/web_interface/templates/plugins.html

```diff
@@ -0,0 +1,38 @@
+{% extends "base.html" %}
```

Comment:
Rendered template looks nice enough. I would like some optical improvements for the exact representation but these can be made later on.